velox/exec/tests/QueryAssertionsTest.cpp

```diff
@@ -365,6 +365,20 @@ TEST_F(QueryAssertionsTest, nullDecimalValue) {
       "Types of expected and actual results do not match");
 }
 
+TEST_F(QueryAssertionsTest, nonNullDecimalValue) {
+  auto test = makeRowVector({
+      makeFlatVector<int64_t>({10, 20}, DECIMAL(12, 2)),
+  });
+  createDuckDbTable({test});
+
+  auto plan = PlanBuilder().values({test}).project({"c0"}).planNode();
+  assertQuery(plan, "SELECT c0 FROM tmp");
+
+  EXPECT_NONFATAL_FAILURE(
```

Comment:
Can we print the values here as well?

Reply:
Print the expected values as a comment??

Reply:
Fixed.

Reply:
No, as part of the assert. Something like
```
assertQuery(plan, "SELECT c0 + 1 FROM tmp"),
      "Expected 2, got 2
2 extra rows, 2 missing rows
2 of extra rows:
	0.10
	0.20

2 of missing rows:
	1.10
	1.20
        ");
```

Reply:
Fixed.